src/map/lua/lua_baseentity.cpp

```diff
@@ -10236,6 +10236,98 @@ void CLuaBaseEntity::recalculateAbilitiesTable()
     PChar->pushPacket(new CCharAbilitiesPacket(PChar));
 }
 
+/************************************************************************
+ *  Function: getEntitiesInRange()
+ *  Purpose : Returns a Lua table of entities within range of the base entity
+ *  Example : local players = npc:getEntitiesInRange(target, aoeType, radiusOrigin, distance, findFlags, validTargets)
+ *  Notes   : if the passed distance is nil or 0 (as well as many other combinations), empty table will be returned
+ *            examples
+ *               aoeType: xi.aoeType.REAR_CONE
+ *               radiusOrigin: xi.aoeRadius.ATTACKER
+ *               findFlags: xi.findFlag.HIT_ALL
+ *               findFlags: xi.findFlag.ALLIANCE | xi.findFlag.PET | xi.findFlag.IGNORE_BATTLEID
+ *               validTargets: xi.targetType.ENEMY | xi.targetType.PLAYER_PARTY
+ ************************************************************************/
+
+sol::table CLuaBaseEntity::getEntitiesInRange(CLuaBaseEntity* PLuaEntityTarget, sol::variadic_args va)
+{
+    auto* baseEntity = dynamic_cast<CBattleEntity*>(m_PBaseEntity);
+    auto  players    = lua.create_table();
```

Comment:
If this is targetfind-based, it can find more than players depending on the flags, no?

Reply:
Good catch, yes. Started from my old binding. I'll update the variable

Reply:
Done